installer/Migration/V5_1_0/permission/api.yaml

```diff
@@ -127,6 +127,19 @@ apiv2_performance_performance_tracker_logs:
     - { role: Admin, permission: { read: true, create: true, update: true, delete: true, self: false } }
     - { role: ESS, permission: { read: true, create: true, update: true, delete: true, self: true } }
 
+apiv2_performance_tracker_reviewers_autocomplete:
+  description: 'API-v2 Performance - Performance Tracker Reviewers Autocomplete'
```

Comment:
there is another API that Sharuka has used for autcomplete in tracker save/edit screens but implemented differently. maybe we can reuse this for that scenario also. didn't go to change with this ticket. Had to put this name because it was conflicting with that API